pages/blog/2020-summary.md

```diff
@@ -0,0 +1,929 @@
+
+<Figure
+  src="/img/posts/2020-summary/star-history-generator.webp"
+  caption="Figure 7: GitHub stars growth in generator repository"
+/>
+
+### Traffic growth
+
+All numbers went up :sweat_smile:
+
+The growth of contributors was not organic but driven by our activities in the [Hacktoberfest](/blog/hacktoberfest-summary-2020) event.
+
+<table>
+  <tr>
+    <td align="center">
+    </td>
+    <td align="center">
+      2019
+    </td>
+    <td align="center">
+      2020
+    </td>
+  </tr>
+  <tr>
+    <td align="center">
+      Contributors
+    </td>
+    <td align="center">
+      76
+    </td>
+    <td align="center">
+      119
+    </td>
+  </tr>
+  <tr>
+    <td align="center">
+      Commits
+    </td>
+    <td align="center">
+      1.5k
+    </td>
+    <td align="center">
+      3.5k
+    </td>
+  </tr>
+  <tr>
+    <td align="center">
+      Pull requests (opened/closed)
+    </td>
+    <td align="center">
+      399/397<br/>99% closed
+    </td>
+    <td align="center">
+      1200/1175<br/>97% closed
+    </td>
+  </tr>
+  <tr>
+    <td align="center">
+      Issues (opened/closed)
+    </td>
+    <td align="center">
+      322/287<br/>89% closed
+    </td>
+    <td align="center">
+      587/401<br/>68% closed
+    </td>
+  </tr>
+</table>
+
+We're closing fewer issues. This number got worse than last year because we freeze progress on the asyncapi repository, and the last few months did not continue any efforts towards specification 2.1 and 3.0 releases. The reason was pretty simple; we were involved in discussions with many players and planning the next step forward to scale up in 2021. It would not be fair with the AsyncAPI community to work on the next specification versions, with the current government process, knowing we prepare [partnership with Postman](/blog/asyncapi-partners-with-postman).
+
+### What's next with GitHub
+
+We are doing well on GitHub, and so far, I don't see any improvements other than the further automation that we have in plans to handle even more traffic next year.
+
+## npm
+
+The majority of our tools are written in JavaScript/TypeScript, and we host them on Node Package Manager(npm)
+
+### Top packages
+
+Our most important packages are [the Generator](https://www.npmjs.com/package/@asyncapi/generator) and [the Parser](https://www.npmjs.com/package/@asyncapi/parser). 
+
+<table>
+  <tr>
+    <td align="center">
+    </td>
+    <td align="center">
+      2019
+    </td>
+    <td align="center">
+      2020
+    </td>
+  </tr>
+  <tr>
+    <td align="center">
+      Generator
+    </td>
+    <td align="center">
+      21 536
+    </td>
+    <td align="center">
+      134 224
+    </td>
+  </tr>
+  <tr>
+    <td align="center">
+      Parser (published May 2019)
+    </td>
+    <td align="center">
+      18 591
+    </td>
+    <td align="center">
+      173 286
+    </td>
+  </tr>
+</table>
+
+The change in numbers is so huge that I checked them twice to make sure I'm not wrong.
```

Comment:
FYI, a big percentage of these downloads are coming from bots.

Reply:
how do you know? why they were not that active last year?

Reply:
I know because every time I was publishing a package (not AsyncAPI related) I was almost instantly having hundreds or thousands of downloads, which is impossible 😄 These numbers are very fuzzy but it's fine, just wanted you to know. As long as you provide both numbers including bots, it's ok.

Reply:
> As long as you provide both numbers including bots, it's ok.

you mean what both numbers? The thing about bots is very relative when I publish a new package I have even several downloads immediately or after release but it is never constant and it is never thousands. And this theory doesn't explain why numbers were so small in 2019

Reply:
whatever this bot traffic is, it doesn't see to be bad and fake https://www.reddit.com/r/javascript/comments/7ya9up/why_are_bots_downloading_npm_packages/

Reply:
Oh no no. I don't mean it explains why numbers were so low in 2019. I mean that a part of it may be bots (crawlers) and that we can't know how much they represent but in my experience, they were the majority (in unknown packages only I was using for myself). What I mean by "both numbers including bots" is that 2019 and 2020 both contain the downloads bots are doing so it's ok. Here's someone asking this same question. It seems to have gotten better over time though: https://www.reddit.com/r/javascript/comments/7ya9up/why_are_bots_downloading_npm_packages/.

Reply:
In short. Don't change anything. Just wanted you to know that it exists 😄

Reply:
yes, and first answer makes a lot of sense and is actually good for us `There are networks segregated from the internet that host their own NPM servers. Some of those downloads are definitely for those types of networks.`

this is how corporations work, they install jfrog artifactory, make a copy of npm on their side, do scans, security, and from time to time update cache. So the presence of those actually indicates numbers should be even higher

Reply:
so we leave it as it is or?